plugins/live-logs.yml

```diff
@@ -0,0 +1,8 @@
+# The live-logs plugin
+pluginName: live-logs
+version: v1.0.0
+repository: https://github.com/jfrog/live-logs
+maintainers:
+  - amithins
+  - eyalbe4
+tag: v1.0.0
```

Comment:
It looks like tag v1.0.0 does not exist on ```https://github.com/jfrog/live-logs```.
This will cause the automatic fail.